pkg/addressbook/doc.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+Package addressbook provides persisted mapping between overlay (topology) address
+and bzz.Address address, which contains underlay (physical) address.
```

Comment:
I would say that underlay contains both physical and p2p addresses.

Reply:
My thinking for this instance was that that reader would follow `bzz.Address` implementation and figure out rest there.
Anyway, I update this comment according to comment.